ui/store/actions.ts

```diff
@@ -3760,21 +3794,32 @@ export function setFirstTimeFlowType(
   };
 }
 
-export function setSelectedSettingsRpcUrl(
-  newRpcUrl: string,
+export function setSelectedNetworkUUID(
```

Comment:
"Network" is a user-facing term, so I think we should either lean into the less technical side or the more technical side. Since this is closer to the user-facing side my vote would be to drop the "UUID" here.

Reply:
I've changed this to `setSelectedNetworkConfigurationId`. Could see it being `setSelectedNetwork` or `setActiveNetworkConfigurationId` or `setActiveNetwork` (though `setActiveNetwork` is the method name on the `NetworkController` for actually changing the provider which we may not want to confuse with this method which is for changing the selectedNetworkConfigurationId in the redux store).